frontend/src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -31,6 +33,8 @@ const CheckboxGroup: FunctionComponent<ComponentProps> = (props) => {
       ? checkboxes?.concat(data.value as string)
       : checkboxes?.filter((checkbox) => checkbox !== data.value);
     addCheckboxes(updatedCheckboxes);
+    props.validateCheckboxOptions &&
+      props.validateCheckboxOptions(data.value as string, data.checked);
```

Comment:
This is where you check if the checkbox is deselected, then call the function. Also, we don't use this kind of syntax when calling functions. You put it in a proper if statement. More readable. It's the recommended syntax for assignment operations, even those involving function calls. Basically, I wouldn't complain if it was:
`const valid = props.validateCheckboxOptions &&
      props.validateCheckboxOptions(data.value, data.checked);`

Reply:
Updated